sections/02_intro.md

```diff
@@ -68,12 +84,44 @@ large. New neural network approaches are also well-suited for addressing
 distinct challenges. For example, neural networks structured as autoencoders or
 as adversarial networks require no labels and are now regularly used for
 unsupervised tasks. In this review, we do not exhaustively discuss the different
-types of deep neural network architectures. A recent book from Goodfellow et al.
+types of deep neural network architectures; an overview of the principal terms used
+herein is given in Table 1. A recent book from Goodfellow et al.
 [@url:http://www.deeplearningbook.org/] covers these in detail. Finally, the
 larger datasets now available are also sufficient for fitting the many
```

Comment:
Already mentioned above.

Reply:
@agapow Thanks for your comments! I made changes in response to them, with the exception of this, since I was a bit confused about what you meant.